CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+### 4.3.1
+
+This release of Teleport contains multiple bug fixes.
+
+* Reverted base OS in container images to Ubuntu. [#4054](https://github.com/gravitational/teleport/issues/4054)
```

Comment:
I feel like this line item is confusing because:
1) We haven't changed the base image for the immutable (`4.3.0`-style) images from `ubuntu`, and the line item makes it sound like we did.
2) We never announced the mutable (`4.3`-style) images in the first place - people just found them independently and didn't like the `alpine` base. We've reached out to those people independently and told them that we've changed that base back to `ubuntu`. Most others probably have no idea that we ever even used an `alpine` base for any time at all, so there's no need to call attention to it - it was only there for about a week.

If anything, this should probably point out that the base image for immutable 4.3.1+ images is now `ubuntu:20.04` to clean up some vulnerabilities (https://github.com/gravitational/teleport/pull/4004).

If we do want to make an announcement about the mutable auto-updating images, we should have done it in the 4.3.0 release notes. We have some details in Teleport docs now as well, and we'll be calling out the option via the Teleport downloads page in the near future. I think doing anything else is just confusing.